packages/governance/src/contractGovernor.js

```diff
@@ -73,7 +80,18 @@ const validateQuestionFromCounter = async (zoe, electorate, voteCounter) => {
  * contract (as electionManager) so clients will be able to look up the state
  * of the governed parameters.
  *
- * @type {ContractStartFn}
+ * @template {object} PF Public facet of governed
+ * @template {ContractPowerfulCreatorFacet} CF Creator facet of governed
+ * @type {ContractStartFn<GovernorPublic, GovernedContractFacetAccess<PF>,{
+ *   timer: Timer,
+ *   electorateInstance: Instance,
+ *   governedContractInstallation: Installation<CF>,
+ *   governed: {
+ *     issuerKeywordRecord: IssuerKeywordRecord,
+ *     terms: {main: {[CONTRACT_ELECTORATE]: unknown}},
```

Comment:
I think this must be an Amount
```suggestion
 *     terms: {main: {[CONTRACT_ELECTORATE]: Amount}},
```